supplementary/alica_ros1/alica_tests/src/test/Expr/src/AdjacentSuccessMasterPlan3254486013443203397.cpp

```diff
@@ -39,6 +39,52 @@ std::shared_ptr<UtilityFunction> UtilityFunction3254486013443203397::getUtilityF
     /*PROTECTED REGION END*/
 }
 
+/**
+ * Transition: 3345031375302716643 (3345031375302716643)
+ *   - Comment:
+ *   - Source2Dest: EntryState --> SecondState
+ *
+ * Precondition: 807250359520655888 (807250359520655888)
+ *   - Enabled: true
+ *   - PluginName: DefaultPlugin
+ *   - ConditionString:
+ *   - Variables:
+ *   - Quantifiers:
+ *
+ * Abstract Plans in EntryState:
+ *   - AdjacentSuccessSubPlan (1682631238618360548)
+ */
+bool PreCondition807250359520655888::evaluate(std::shared_ptr<RunningPlan> rp, const IAlicaWorldModel* wm)
+{
+    /*PROTECTED REGION ID(3345031375302716643) ENABLED START*/
+    std::cout << "The PreCondition 807250359520655888 in Transition '3345031375302716643' is not implement yet!" << std::endl;
```

Comment:
Not implemented is intentional?

Reply:
Yes, all of the plans in alica use the new conditions. Only for unit testing some plans need to implement the old conditions.